HPXMLtoOpenStudio/measure.rb

```diff
@@ -2013,6 +2015,16 @@ def self.add_generators(runner, model)
     end
   end
 
+  def self.add_batteries(runner, model, spaces)
+    @hpxml.batteries.each do |battery|
```

Comment:
Have you tested that multiple batteries are handled correctly?

Reply:
@shorowit I'll add tests for this case.

Question:
* How do we handle the case of multiple batteries and a PV system? 2 ELCDs, 1 of which has an inverter? Which battery goes on the ELCD with the inverter?

Reply:
We are going to limit the number of batteries to 1, using EPvalidator.xml.